Formula/r/rsjpg.rb

```diff
@@ -0,0 +1,17 @@
+class Rsjpg < Formula
+  desc "Tool to combine multiple images into a single image"
+  homepage "https://github.com/steelPiber/rsjpg"
+  url "https://github.com/steelPiber/rsjpg/archive/refs/tags/v0.0.2.tar.gz"
+  sha256 "84a661ccc6a39ece8884dc35bdda60be5ccde76d856487580c7641d562d980db"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/rsjpg", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.